Tasks/AzureRmWebAppDeploymentV4/deploymentProvider/BuiltInLinuxWebAppDeploymentProvider.ts

```diff
@@ -94,8 +99,17 @@ export class BuiltInLinuxWebAppDeploymentProvider extends AzureRmWebAppDeploymen
 
     public async UpdateDeploymentStatus(isDeploymentSuccess: boolean) {
         if(this.kuduServiceUtility) {
-            await super.UpdateDeploymentStatus(isDeploymentSuccess);
-            if(this.zipDeploymentID && this.activeDeploymentID && isDeploymentSuccess) {
+           
+ 	        this.activeDeploymentID =  this.kuduServiceUtility.getDeploymentID();
+            if(isDeploymentSuccess == false){
+                await super.UpdateDeploymentStatus(isDeploymentSuccess);
+            }
+            else if(this.zipDeploymentID && this.activeDeploymentID && isDeploymentSuccess) {
+                await addReleaseAnnotation(this.azureEndpoint, this.appService, isDeploymentSuccess);
+                let appServiceApplicationUrl: string = await this.appServiceUtility.getApplicationURL(!this.taskParams.isLinuxApp 
+                    ? this.taskParams.VirtualApplication : null);
+                console.log(tl.loc('AppServiceApplicationURL', appServiceApplicationUrl));
```

Comment:
Is this a log for `debug`? The other logs have used "`tl.debug`", why did you prefer to use `console.log` here?

Reply:
It was same in [AzureRmWebAppDeploymentProvider.ts](https://github.com/microsoft/azure-pipelines-tasks/blob/59c77a6d7f7a75d52d16672f045a2a430e8902fb/Tasks/AzureRmWebAppDeploymentV4/deploymentProvider/AzureRmWebAppDeploymentProvider.ts). 

So, I'm closing this comment.